tools/loomexec/src/main.cpp

```diff
@@ -108,8 +109,18 @@ static void initialize(int argc, const char **argv)
 
     assemblyPath = "";
 
+    int argStart = 1;
```

Comment:
Yes, namely I added `--ignore-missing-types` and `--verbose`. We might want to use a small lib that handles parsing of arguments across `LoomDemo` and `loomexec` at some point, but we only need simple parsing for now.